workers/unity/Packages/com.improbable.gdk.core/Components/ISnapshottable.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Unity.Entities;
+
+namespace Improbable.Gdk.Core
```

Comment:
Does this definitely belong the the `Core`?

Reply:
Yes - it makes the namespace of `ISpatialComponentData`, `ISpatialComponentUpdate`, and `ISpatialComponentSnapshot`